INSTALL.md

```diff
@@ -68,6 +68,24 @@ If you need to find this info again later, you can run:
 brew info elasticsearch
 ```
 
+### MYSQL Database
+Ensure your databse is running.
+```
+mysql.server start
+```
+
+Then run mysql creation script from root directory
+```
+./create-mysql-db.sh
+```
+If you would like to have a custom db setup then you can pass in the necessary args.
+*NOTE* be sure to update your local settings `local.py` to account for these changes.
```

Comment:
`> *NOTE:* Be sure to update your local settings `local.py` to account for these changes.
    ```
    ./create-mysql-db.sh <dbname> <username> <password>
    ```

For consistency with other readme notes.